### PR TITLE
qa/suites/orch/cephadm: add 8.stream + container_tools

### PR DESCRIPTION
**Pull request by liewegas**
Signed-off-by: Sage Weil <sage@newdream.net>

### CODE REVIEW

**Comment by sebastian-philipp** [comment]
this is a follow-up on #43813

**Comment by liewegas** [comment]
jenkins test api

File: qa/distros/podman/centos_8.stream_container_tools.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  selinux:
+    whitelist:
+      - scontext=system_u:system_r:logrotate_t:s0
+
+tasks:
+- pexec:
+    all:
+    - sudo cp /etc/containers/registries.conf /etc/containers/registries.conf.backup
+    - sudo dnf -y  module reset container-tools
+    - sudo dnf -y  module install container-tools
```
Comment (fultonj) [inline qa/distros/podman/centos_8.stream_container_tools.yaml:13]: LGTM.
I tried this on my CentOS Stream release 8 system and got podman version 3.4.1-dev. 
In TripleO we're still enabling container-tools:3.0 [1] and getting podman version 3.0.2-dev. 
If this works for you, I think it's good ceph will be ahead. I expect TripleO will catch up.

[1] https://github.com/openstack/tripleo-quickstart/blob/master/config/release/centosci/master-current-tripleo.yml#L69-L70

**Comment by liewegas** [comment]
Sigh, got
```
2021-11-09T23:49:45.820 INFO:teuthology.orchestra.run.smithi101.stderr:stat: stderr Error: OCI runtime error: container_linux.go:380: starting container process caused: process_linux.go:385: applying cgroup configuration for process caused: failed to write 34644 to cgroup.procs: write /sys/fs/cgroup/memory/machine.slice/libpod-8210bc530cf009eda2680a085fcf9771ca31d035e6a5fe3107fd91b09da42b31.scope/cgroup.procs: no such device
```
with podman 3.4.1-dev

/a/sage-2021-11-09_23:13:55-orch:cephadm-wip-sage2-testing-2021-11-09-1019-distro-basic-smithi/6494794

**Pull request closed without merging**